### ISSUE TITLE
Base64 encoded certificates are not uploaded properly

### ISSUE THREAD

**Issue by dstokesf5** [F5Networks/f5-appsvcs-extension#580]
### Environment
 * Application Services Version: 3.35.0
 * BIG-IP Version: 16.1.2

### Summary
While the schema allows using the `base64` property for `Certificate.certificate` and `Certificate.privateKey`, the files that are uploaded to the BIG-IP are still encoded and therefore invalid.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "ADC",
    "schemaVersion": "3.0.0",
    "Tenant": {
        "class": "Tenant",
        "Application": {
            "class": "Application",
            "webcert": {
                "class": "Certificate",
                "certificate": {
                    "base64": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNuRENDQWdXZ0F3SUJBZ0lKQUo1bjJiME9DRWp3TUEwR0NTcUdTSWIzRFFFQkN3VUFNR2N4Q3pBSkJnTlZCQVlUQWxWVE1STXdFUVlEVlFRSURBcFhZWE5vYVc1bmRHOXVNUkF3RGdZRFZRUUhEQWRUWldGMGRHeGxNUlF3RWdZRFZRUUtEQXRtTlY5T1pYUjNiM0pyY3pFYk1Ca0dBMVVFQXd3U2MyRnRjR3hsTG1WNFlXMXdiR1V1Ym1WME1CNFhEVEUzTVRFeU5qRTVOVEF5TkZvWERURTRNREl5TlRFNU5UQXlORm93WnpFTE1Ba0dBMVVFQmhNQ1ZWTXhFekFSQmdOVkJBZ01DbGRoYzJocGJtZDBiMjR4RURBT0JnTlZCQWNNQjFObFlYUjBiR1V4RkRBU0JnTlZCQW9NQzJZMVgwNWxkSGR2Y210ek1Sc3dHUVlEVlFRRERCSnpZVzF3YkdVdVpYaGhiWEJzWlM1dVpYUXdnWjh3RFFZSktvWklodmNOQVFFQkJRQURnWTBBTUlHSkFvR0JBTEVzdVhtU1hWUXBZanJaUFcrV2lUQmpuNDkxbXdaWVQ3UTkyVjFIbFNCdE02V2RXbEsxYVpONXNvdmZLdE9YN1lybTh4YStlNG8vekoyUVlMeXl2NU8rdDJFR04vNHFVRWpFQVBZOW13SmRmelJReTZIeXptODRKMFFrVHVVSi9Fak51UGppM0QwUUpSQUxVVHp1MVVxcURDRXRpTjlPR3lYRWtoN3V2YjdCQWdNQkFBR2pVREJPTUIwR0ExVWREZ1FXQkJTVkhQTnJHV3JqV3ladmNrUXhGWVdPNTlGUkZqQWZCZ05WSFNNRUdEQVdnQlNWSFBOckdXcmpXeVp2Y2tReEZZV081OUZSRmpBTUJnTlZIUk1FQlRBREFRSC9NQTBHQ1NxR1NJYjNEUUVCQ3dVQUE0R0JBSmVKOVNFY2tFd1Boa1hPbStJdXFmYlVTL1JjemlpZkJDVG1WeUUrRmEvajlwS1NZVGdpRUJOZGJKZUJFYStnUE1sUXRiVjdZMmR5OFRLeC84YXhWQkhpWEM1Z2VETUw3Y2F4T3JBeUhZQnBueDY5MHhKVGg1T0lPUkJCTS9hL052YVIrUDNDb1ZlYnIvTlBSaDlvUk54bm50bnF2cUQ3U1cwVTNaUGUzdEpjCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K"
                },
                "privateKey": {
                    "base64": "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpQcm9jLVR5cGU6IDQsRU5DUllQVEVECkRFSy1JbmZvOiBBRVMtMjU2LUNCQyxEOEZGQ0U2QjI1NTYwMTU4N0NCNTRFQzI5QjczN0QzMQoKa3Y0RmMzSm4wVWprajB5Ump0K2dRUWZCTFNORjJhUkxVRU5YbmxyN1hwenF1MEFocjNqUzFiQUFuZDhJV25zUgp5SUxxVm1Lc1lGMkRvSGgwdFdpRUFRNy95L2ZlNURURmhLN040V21sNmtwMnlWTWtQNktDNHNzeVlQdzI3a2pLCkRCd0JaNU84SW9lajA4QTVzZ3NMQ21nbGJtdFNQSEpVbjE0cFFuTVRtTE9wRXRPc3U2UysyaWJQZ1NOcGRnMGIKQ0FKTkcvS0hlK1ZreDU5cU5EeURlS2I3RlpPbHNYMzAreTY3elVxOUdRcUpFRHV5c1BKMkJVTlAwSUpYQWpzdApGSXQxcU5vWmV3KzVLRFlzN3UvbFB4Y01HVGlyVWhnSTg0Snk0V2NEdlNPc1AvdEtseGowNFRiSUUzZXBtU0t5CitUaWhIa3dZN25nSUd0Y20zU2ZxazVqejJSWG9qMS9BYzNTVzhrVlRZYU9Vb2dCaG43ekFxNFdqdTZFdDRoUUcKUkdhcHNKcDFhQ2VaL2E0UkNEVHhzcGNLb01hUmE5Ny9VUlFiMGhCUkd4M0RHVWh6cG1YOXpsN0pJMlhhNUQzUgptZEJYdGpMS1lKVGRJTWRkMjdwckJFS2hNVXBhZTJyejVNdzRKOTA3d1plQnEvd3UrenA4TEFuZWNmVGUybkdZCkUzMngxVTdnU0VkWU9HcW53eHNPZXhiMWpLZ0NhNjdOdzlUbWNNUFY4em1IN1I5cWR2Z3hBYkF0d0JsMUY5T1MKZmNHYUM3ZXBmMUFqSkx0YVg3a3JXbXpnQVNIbDI4WW5oOWxtR01kdis1UVlNWnZLRzBMT2cvbjNtOHVKNnNLeQpJenp2YUpzd3duMGo1UDUrY3p5b1Y1Q3Z2ZENmS25OYiszalVFTjhJMFBQd2pCR0tyNEIxb2p3aG9nVE0yNDhWCkhSNjlENlR4RlZNZkdweUpoQ1BrYkdFR2JwRXBjZmZwZ0t1Qy9tRXRNcXlEUVhKTmFWNUhPNkhnQUo5RjFQNnYKNWVoSEhUTVJ2ekNDRml3bmRIZGxNWFVqcVNOand3Nm1lNmRyNkxpQVBiZWpkemhMMnZXeDFZcWViT2N3UXgzRwotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo="
                },
                "passphrase": {
                    "ciphertext": "ZjVmNQ==",
                    "protected": "eyJhbGciOiJkaXIiLCJlbmMiOiJub25lIn0"
                }
            }
        }
    }
}
```

2. Observe the following error response:
```json
        {
            "code": 422,
            "message": "declaration failed",
            "response": "01070712:3: unable to validate certificate, invalid x509 file (/Tenant/Application/webcert.crt).",
            "host": "localhost",
            "tenant": "Tenant",
            "runTime": 1198
        }
----
        {
            "code": 422,
            "message": "declaration failed",
            "response": "01070712:3: Unable to verify key (/Tenant/Application/webcert.key) is protected by provided passphrase.",
            "host": "localhost",
            "tenant": "Tenant",
            "runTime": 1431
        }
```

### Expected Behavior
Certificate and keys should be uploaded to the BIG-IP with the appropriate format/encoding

### Actual Behavior
Certificate and keys are used "as-is" and cause errors.

**Comment by dstokesf5** [comment]
This issue is in our internal backlog as AUTOTOOL-3068.

**Comment by ghalevy** [comment]
@dstokesf5 - any update on this?  

We have moved to `pkcs12` as it does allow base64 encoding - however we are having issues with `pkcs12` and would like to revert to `x509`.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.38.0 release of AS3.
